Assets/Scripts/Tutorial/LivingRoomTutorial.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LivingRoomTutorial : Tutorial
+{
+    public override bool Completed
+    {
+        get
+        {
+            return PlayerPrefsUtil.CompletedLivingRoomTutorial;
+        }
+    }
+
+    [SerializeField]
```

Comment:
There's an easy struct that can replace this:

1. Create new class that's "TutorialObject"
2. Class has a static dict by obj name of <string, TutorialObject>
3. Pull refs from that static dict using a static method
4. Creates baseline for a more sustainable method than this